package.json

```diff
@@ -895,6 +895,7 @@
     "blurhash": "^2.0.1",
     "brace": "0.11.1",
     "brok": "^5.0.2",
+    "buildkite-test-collector": "^1.6.5",
```

Comment:
@dkirchan @delanni just stumbled upon these new dependencies, is there any reason we made them production (`dependencies` section) and not dev-only (`devDependencies` section)?

Reply:
Thanks for confirming @delanni! @dkirchan would you mind moving these dependencies to `devDependencies` when you have a chance?

Reply:
Hello there. @azasypkin I will do tomorrow morning that I ll be back. First steps in the kibana repo so still missing too many things.

Reply:
No worries at all @dkirchan, thanks!